pysolarfocus/components/base/component.py

```diff
@@ -113,6 +120,14 @@ def __get_holding_values(self)->list[tuple[str,DataValue]]:
             self.__holding_values = sorted(self.__holding_values,key=lambda item: item[1].address)
         return self.__holding_values
     
+    def __get_performance_calculators(self)->list[tuple[str,PerformanceCalculator]]:
```

Comment:
This should probably be called in the initialize method, atm there is no obvious way to initialize this without calling `has_performance_calculators`